Modules/Sources/DesignSystem/Typography/FontManager.swift

```diff
@@ -39,7 +39,9 @@ extension UIFont {
         return makeFont(with: descriptor)
     }
 
-    static func makeFont(with descriptor: FontDescriptor) -> UIFont {
+    private static func makeFont(with descriptor: FontDescriptor) -> UIFont {
+        FontManager.registerCustomFonts()
```

Comment:
This makes it easier to use fonts from previews. `registerCustomFonts` only performs the work once.